packages/ilios-common/addon/components/yup-validation-message.hbs

```diff
@@ -0,0 +1,5 @@
+{{#each this.messages as |m|}}
+  <span class="validation-error-message" ...attributes>
```

Comment:
why the splattributes here?

Reply:
Just in case. You might toss a different class or a `data-FOO` on there and it will get pushed into every single one. I dunno if we'll use it, I just like to add them so we don't have to re-add them later.

Reply:
i accept this.